beater/sophoscentralbeat.go

```diff
@@ -189,7 +197,7 @@ func LegacyEventEntityToCommonMap(entity sophoscentral.LegacyEventEntity) (commo
 }
 
 //GetSophosAlerts : call alerts API
-func GetSophosAlerts(scb Sophoscentralbeat) (int64, error) {
+func GetSophosAlerts(scb Sophoscentralbeat) error {
 
 	var receivedAlertsLogCount int64
 	var allAlertsLogs int64

```

Comment:
@rajanjoshi18 No need of this function if logsReceivedInCycle just near to publishing received logs